packages/zwave-js/src/lib/commandclass/ThermostatSetpointCC.ts

```diff
@@ -629,11 +629,9 @@ export class ThermostatSetpointCCSet extends ThermostatSetpointCC {
 
 	public serialize(): Buffer {
 		// If a config file overwrites how the float should be encoded, use that information
-		const override =
-			this.getNodeUnsafe()?.deviceConfig?.compat?.overrideFloatEncoding;
 		this.payload = Buffer.concat([
 			Buffer.from([this.setpointType & 0b1111]),
-			encodeFloatWithScale(this.value, this.scale, override),
+			encodeFloatWithScale(this.value, this.scale),
```

Comment:
Uh, this was supposed to be here, but not in the Humidity CCs

Reply:
oops! I didn't mean to edit this one